transport/http/src/test/java/org/kaazing/gateway/transport/http/specification/httpxe/RequestsIT.java

```diff
@@ -0,0 +1,180 @@
+/**
+ * Copyright 2007-2015, Kaazing Corporation. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.kaazing.gateway.transport.http.specification.httpxe;
+
+import org.apache.mina.core.service.IoHandler;
+import org.junit.Ignore;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TestRule;
+import org.kaazing.gateway.resource.address.ResourceAddress;
+import org.kaazing.gateway.resource.address.ResourceAddressFactory;
+import org.kaazing.gateway.transport.IoHandlerAdapter;
+import org.kaazing.gateway.transport.http.HttpAcceptSession;
+import org.kaazing.gateway.transport.http.HttpAcceptorRule;
+import org.kaazing.gateway.transport.http.HttpStatus;
+import org.kaazing.k3po.junit.annotation.Specification;
+import org.kaazing.k3po.junit.rules.K3poRule;
+
+import java.net.URI;
+
+import static org.kaazing.test.util.ITUtil.createRuleChain;
+
+/**
+ * Defines how httpxe will deal with http methods.
+ */
+public class RequestsIT {
+
```

Comment:
I think we should be verifying that the HttpSession has the correct method, headers, etc. For example you could use a mock object instead of the concrete object for the acceptHandler with a matches expectation on HttpSession. Each test may need its own separate expectations (or at least a special matcher to verify the expected method and headers).